angular/Barcode/Getting-Started.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Getting Started for Angular Barcode
-description: Barcode
-platform: Angular
+title: Getting Started with Angular Barcode Component | Syncfusion
+description: Checkout and learn about Getting started with Angular Barcode component of Syncfusion Essential JS 2 and more details.
```

Comment:
Learn here all about getting started with Syncfusion Essential Angular Barcode control, its elements, and more details.

Reply:
Updated as proposed